blog/on_retros/README.md

```diff
@@ -0,0 +1,272 @@
+
+Some things that worked well for us were
+1) Decided that instead of requiring signoff from "the SDET," instead any engineer who didn't make a commit could signoff, but the SDET would work with the engineers to teach them what the standards were for testing.
+2) Automated testing was sufficient for signoff, so we pushed hard on TDD and including automated testing in the "definition of done" for tickets. This led to a better understanding of testing amongst engineers and as a side effect, better code! [Easy to test code is usually well architected.](../an_opinionated_set_of_testing_patterns/README.md)
+
+### Too many meetings
+A common complaint is that engineers have too many meetings to actually have time to do dev work. In order to address this problem, we usually try to figure out what kinds of meetings these are
+
+If they are internal team meetings, like sprint ceremonies, these can be gradually shortened and optimized over time as the team builds competency and mastery of its own practices. For instance, once a team gets good at continuous grooming, it no longer needs a recurring grooming meeting. Once the backlog is in good shape, planning meetings take around 15 minutes. Additionally, since the team owns these meetings, they can just cancel them if they feel it delivers less value than just having more dev time.
+
+If they are external team meetings, like working groups or TAGs or even just checkins on programs or meetings set up to talk about issues that other teams need help for, that's a lot trickier to solve.
+
+I've seen some people try "no meetings Wednesdays" or stuff like that, but it always seems to fall apart after a few weeks. And honestly, I'm not sure if that does anything but fill up the other days with Wednesday's meetings.
+
+What I have been experimenting with, to some successful, is to have the team block off an agreed upon block to "be available" together for dev work. If possible, I would make that time block auto-decline meetings. If its actually important, the organizer will reach out to ask if you can join. This way team can reclaim an hour or two to all be present and doing work together each day. You can tweek as you like over the course of multiple retros.
+
+That being said, I think "meeting culture" deserves its own blog post. Meetings are definitely a useful tool, but in my opinion, maybe like 25% of the meetings I go to are actually good uses of synchronous time being blocked off. And once you start having a bunch of those meetings on repeat every week, the problem snowballs. If everyone is in so many meetings you can't schedule time with the people you need to talk to, when you do get a time, it starts to make sense to schedule recurring "checkins" with people just so you can talk to them at all. The problem compounds rapidly.
```

Comment:
Yes I have a lot of thoughts here - let's compare!

Reply:
for sure